src/components/Header/BaseHeaderMenuItem.jsx

```diff
@@ -47,10 +59,12 @@ BaseHeaderMenuItem.propTypes = {
   current: PropTypes.bool,
   /** The navigation url for the item */
   itemUrl: PropTypes.string,
+  /** Optional onClick handler for itemUrl anchor tag */
+  onClick: PropTypes.func,
   /** Menu item is part of a sub menu */
   subMenu: PropTypes.bool,
   /** [Developer] Adds data-testId attributes for component testing */
-  testMode: PropTypes.bool
+  testMode: PropTypes.bool,
```

Comment:
Many of the changes in this file are due to Prettier. The only substantive change is adding the `onClick` property.